Jenkinsfile

```diff
@@ -42,10 +42,12 @@ node('vetsgov-general-purpose') {
         },
 
         unit: {
-          dockerContainer.inside(commonStages.DOCKER_ARGS) {
-            sh "/cc-test-reporter before-build"
-            sh "cd /application && npm --no-color run test:unit -- --coverage"
-            sh "cd /application && /cc-test-reporter after-build -r fe4a84c212da79d7bb849d877649138a9ff0dbbef98e7a84881c97e1659a2e24"
+          retry(3) {
```

Comment:
@alexandec didn't we do something where unit test retries were set to 5? I don't remember where that was

Reply:
yes, we [increased mocha retries](https://github.com/department-of-veterans-affairs/vets-website/pull/15467/files) from 3 to 5. assuming that is working, this would give us 15 total retries for unit tests. if a flaky test isn't working after that, it probably never will 😂

Reply:
> one thing that might be worth testing here is how long a retry takes with this method vs using the mocha `retries` flag

🤔  That's a good idea. @alexandec, did y'all have to measure the number of times people were re-running tests? Or did y'all just make a change and hope it helped.

Reply:
Definitely the latter, there was no measurement involved 🤞 Also I'm not sure if there's any output to show a retry is happening, or if it just does it silently

Reply:
The Jenkins `retry` directive seems to display when a command is run again after an earlier failure. 

For example: http://jenkins.vfs.va.gov/blue/organizations/jenkins/testing%2Fvets-website/detail/wrap-tests-in-retry/5/pipeline/

![image](https://user-images.githubusercontent.com/6130520/110530089-a5a44280-80df-11eb-8cd6-650c22e0f568.png)

Reply:
That's useful because then we can see how often retries are used and determine whether to adjust the number of retries in the future. Not sure if we get that kind of output with the mocha flag.